components/butterfly/Cargo.toml

```diff
@@ -47,4 +46,4 @@ tempfile = "*"
 [features]
 default = ["protocols"]
 protocols = []
-deadlock_detection = ["parking_lot/nightly", "parking_lot/deadlock_detection"]
+ignore_inconsistent_tests = []
```

Comment:
There's no need to add `ignore_inconsistent_tests` back, it wasn't part of https://github.com/habitat-sh/habitat/commit/e57c2abb53cd79a20d6fe97b0758b9b0d9df5b3b

Reply:
That's odd.  I'm guessing this was part of one of the revert conflicts, and apparently I resolved it incorrectly.  😁